src/plugins/intel_cpu/src/utils/jit_kernel.hpp

```diff
@@ -632,6 +923,22 @@ void boolean_expression<T>::cmp(const Xbyak::Label & exit) const {
             _kernel.je(exit, Xbyak::CodeGenerator::T_NEAR);
```

Comment:
BTW shouldn't it be `_kernel.jne` here? If not some comment required for sure.

Reply:
This jump is used to go to the exit point.
This is used to generate the following code:
> cmp(a, b);
je(else);
/// here a != b
L(else);
/// here a == b

Reply:
I guess we need to add comment that. Right now it looks a little bit confusing.